multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/grouple/GroupLeGenerator.kt

```diff
@@ -9,12 +9,12 @@ class GroupLeGenerator : ThemeSourceGenerator {
 
     override val themeClass = "GroupLe"
 
-    override val baseVersionCode = 19
+    override val baseVersionCode = 20
 
     override val sources = listOf(
         SingleLang("ReadManga", "https://readmanga.live", "ru", overrideVersionCode = 46),
-        SingleLang("MintManga", "https://mintmanga.live", "ru", overrideVersionCode = 46),
-        SingleLang("AllHentai", "https://2023.allhen.online", "ru", isNsfw = true, overrideVersionCode = 22),
+        SingleLang("MintManga", "https://mintmanga.com", "ru", overrideVersionCode = 46),
```

Comment:
```suggestion
        SingleLang("MintManga", "https://mintmanga.com", "ru", isNsfw = true, overrideVersionCode = 46),
```
https://mintmanga.com/list/limitation/r18_

Reply:
No Hentai source.

Reply:
even if it's not mainly hentai, it's still NSFW https://mintmanga.com/persikovyi_sorbet__A35c96/vol1/3?mtr=1